src/Hl7.Fhir.Base/Serialization/engine/PocoSerializationEngine_Json.cs

```diff
@@ -0,0 +1,68 @@
+#nullable enable
+
+using Hl7.Fhir.Introspection;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Utility;
+using System;
+using System.Linq;
+using System.Text.Json;
+
+namespace Hl7.Fhir.Serialization;
+
+internal partial class PocoSerializationEngine
+{
+    private readonly FhirJsonPocoDeserializerSettings? _jsonDeserializerSettings;
+    private readonly FhirJsonPocoSerializerSettings? _jsonSerializerSettings;
+    
+    private BaseFhirJsonPocoDeserializer? _jsonDeserializer;
+    private BaseFhirJsonPocoSerializer? _jsonSerializer;
+    
+    private BaseFhirJsonPocoDeserializer getJsonDeserializer() => 
+        _jsonDeserializer ??= new BaseFhirJsonPocoDeserializer(_inspector, _jsonDeserializerSettings!);
```

Comment:
Ha! A Null coalescing assignment combined with the fact that assignment can be used as an expression, so that is can be returned. You're sure you did not use C# before?